index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> “Hello World, we are software developers!”</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <main id="container">
+      <header>
+        <h1>Wangarĩ Muta Maathai</h1>
+        <h2>THE CHAMPION OF THE GREEN WORLD</h2>
+        <img class="image" src="https://www.nobelprize.org/uploads/2018/06/04_05_oslo-1.jpg" alt="Dr. Wangari Maathai receiving her nobel Prize">
+        <p>Wangari Maathai with her Nobel Medal and Diploma at the Nobel Peace Prize Award Ceremony,Oslo City Hall,<br>
+          Norway.The Kenyan ecologist is the first African woman awarded the Nobel peace Prize.
+        </p>
+      </header>
+      <section class="tribute-info">
+        <h3 class="headline">Here's a time line of Dr. Wangari Maathi's life:</h3>
+        <ul class="list_items">
+          <li><b>1940</b>-Nyeri, Kenya</li>
+          <li><b>1964</b>-Obtained a degree in Biological Sciences from Mount St. Scholastica College in Atchison, Kansas</li>
+          <li><b>1966</b>-Earned a Master of Science degree from the University of Pittsburgh.</li>
+          <li><b>1969</b>-Gets married to Mwangi Mathai.</li>
+          <li><b>1971</b>-Obtained a Ph.D. from the University of Nairobi.</li>
+          <li><b>1977</b>-She establishes the Green Belt Movement (GBM) which an environmental organization that empowers communities,
+             particularly women, to conserve the environment and improve livelihoods
+          </li>
+          <li><b>1979</b>-Gets divorced to her husband,who the had 3 children together.</li>
+          <li><b>1982</b>-Tried contesting for the vacant Nyeri parliament seat but was disqualified on technicality by the court.</li>
+          <li><b>1989</b>-Opposes the construction of a  60 storey building in Uhuru Park.</li>
+          <li><b>1991</b>-she received the Goldman Environmental Prize in San Francisco and the Hunger Project's Africa Prize for Leadership in London.</li>
+          <li><b>1997</b>-Vied for the presidential seat on Lieral Party of Kenya but lost to the incubent president Moi.</li>
+          <li><b>2003</b>-she was appointed Assistant Minister in the Ministry for Environment and Natural Resources and served in that capacity until November 2005.</li>
+          <li><b>2004</b> - was awarded the 2004 Nobel Peace Prize for her "contribution to sustainable development, democracy and peace.</li>
+          <li><b>2011</b> - Died of complications arising from ovarian cancer while receiving treatment at a Nairobi hospital</li>
+        </ul>
+        <h3 class="headline">If you have time, you should read more about this champion of environment at
+          <a class="tribute-link" href="https://en.wikipedia.org/wiki/Wangari_Maathai" rel="noreferrer">Wikipedia</a>
+        </h3>
+      </section>
+    </main>
+  </body>
+</html>
```

Comment:
- [ ] good work and keep up the good work of using semantics well. I have nothing more to add apart from congratulations and best of luck in future projects